app/views/admin/_navigation.html.erb

```diff
@@ -1,13 +1,11 @@
 <ul class="admin_nav">
   <li><%= link_to 'Users', admin_users_path %></li>
+  <li><%= link_to ' \_ Deleted Users', admin_users_path({deleted: true}) %></li>
   <li><%= link_to 'Tracks', admin_assets_path %></li>
-  <li>Comments</li>
-    <ul>
-      <li><%= link_to 'All',  admin_comments_path %></li>
-      <li><%= link_to 'Spam',  admin_comments_path({filter_by: {is_spam: true}}) %> </li>
-      <li><%= link_to 'Not Spam', admin_comments_path({filter_by: {is_spam: false}}) %></li>
-    </ul>
-    <li><%= link_to "Secretz", "/secretz" %></li>
-    <li><%= link_to "Spam Stats", "https://akismet.com/web/1.0/user-stats.php?blog=alonetone.com&api_key=#{Rails.configuration.alonetone.rakismet_key}"%></li>
-    <li><%= link_to "Job Status", "/sidekiq" %></li>
+  <li><%= link_to 'Comments',  admin_comments_path %></li>
```

Comment:
We probably want to keep the <ul> within <ul> structure, but @ofsound knows for sure, he's styling it right now.

Reply:
+1